apps/block_scout_web/lib/block_scout_web/controllers/api/v2/proxy/metadata_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule BlockScoutWeb.API.V2.Proxy.MetadataController do
+  @moduledoc """
+  Controller for the metadata service
+  """
+  use BlockScoutWeb, :controller
+
+  alias Explorer.MicroserviceInterfaces.Metadata
+
+  action_fallback(BlockScoutWeb.API.V2.FallbackController)
+
+  def addresses(conn, params) do
```

Comment:
Maybe add some short doc, like we do in other controllers? 

```suggestion
  @doc """
    Function to handle GET requests to `/api/v2/metadata/addresses` endpoint.
    Some short desc of what it does.
  """
  def addresses(conn, params) do
```